aspnetcore/index.md

```diff
@@ -38,7 +38,7 @@ ASP.NET Core provides the following benefits:
 * Ability to build and run on Windows, macOS, and Linux.
 * Open-source and [community-focused](https://live.asp.net/).
 
-ASP.NET Core ships entirely as [NuGet](https://www.nuget.org/) packages. This allows you to optimize your app to include only the necessary NuGet packages. In fact, ASP.NET Core 2.x apps targeting .NET Core only require a [single NuGet package](xref:fundamentals/metapackage). The benefits of a smaller app surface area include tighter security, reduced servicing, and improved performance.
+ASP.NET Core ships entirely as [NuGet](https://www.nuget.org/) packages. Using NuGet packages allows you to optimize your app to include only the necessary NuGet packages. In fact, ASP.NET Core 2.x apps targeting .NET Core only require a [single NuGet package](xref:fundamentals/metapackage). The benefits of a smaller app surface area include tighter security, reduced servicing, and improved performance.
```

Comment:
Let's keep an eye on this 3rd sentence. ASP.NET Core 2.1 may introduce metapackage support for apps targeting .NET Framework as well. See https://github.com/aspnet/MetaPackages/issues/255#issuecomment-362163031.